app/models/claim_update.rb

```diff
@@ -37,13 +54,15 @@ def send_confirmation_email
     ClaimMailer.submitted(claim).deliver_later
   end
 
-  def update_claim_school_dependent_attributes
-    if claim.claim_school_id_changed?
-      claim.employment_status = nil
+  def reset_dependent_answers
+    DEPENDENT_ANSWERS.each do |attribute_name, dependent_attribute_name|
+      if claim.changed.include?(attribute_name)
+        claim.attributes = {dependent_attribute_name => nil}
+      end
     end
   end
 
-  def update_employment_status_dependent_attributes
+  def determine_inferred_current_school
```

Comment:
Can this just be `infer_current_school`?

Reply:
👍